airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_core.py

```diff
@@ -405,11 +405,11 @@ def test_date_pattern(self, actual_connector_spec: ConnectorSpecification, detai
             property_definition = schema_helper.get_parent(format_path)
             pattern = property_definition.get("pattern")
             if format == "date" and not pattern == DATE_PATTERN:
-                detailed_logger.warning(
+                pytest.fail(
```

Comment:
Maybe I'm misunderstanding CATs, but shouldn't e.g. `source-amazon-ads` fail here, because it has a `start_date` field with `"format": "date"` and `"pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"`?

I'm surprised we have `DATE_PATTERN` as `DD-MM-YYYY` here, because most sources seem to use `YYYY-MM-DD`, which is also what I would expect, because it matches standards like ISO 8601.

Reply:
That's true, it did fail.

Looking more closely at the test, it feels like something that was thrown in to catch something that only impacted the UI. It will need to be modified if we want it to fail iff it would have failed in the platform. I'm thinking that maybe the thing to do is look more closely at the difference in validators to see why ours didn't catch the problem before the config went to platform. Will do that and circle back.